swagger/docs.go

```diff
@@ -1030,12 +1168,12 @@ const docTemplate = `{
 
 // SwaggerInfo holds exported Swagger Info so clients can modify it
 var SwaggerInfo = &swag.Spec{
-	Version:          "v1.1.0-dev",
```

Comment:
These should not be changed. They are read from routes.go from what I can tell. Why are they gone now?

Reply:
I regenerated the docs and now they appear, I think I accidentally deleted the line

Reply:
I think it has something to do with sending multiple files as parameters to the swagger command in the makefile. But since it's working now, let's mark this as resolved.